src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -241,6 +245,15 @@ const ProfilePageContent = ({
 
   const setEditProfileOpen = useSetEditProfileOpen();
   const { updateBackground } = useApiWithProfile();
+  const navigate = useNavigate();
+  const { selectAndFetchCircle } = useApiBase();
+
+  const goToCircleHistory = (id: number, path: string) => {
+    selectAndFetchCircle(id).then(() => {
```

Comment:
You actually want to use selectCircle here instead (I made the same mistake myself and just fixed it in #800)

Reply:
good to know, fixed now